### PR TITLE
creating the user class

### PR DESCRIPTION
**Pull request by Theideabased**
this is the user class i was to create with empty string of all the element

### CODE REVIEW

**Comment by KayFrimp** [comment]
Cohort is not a string... Kindly correct that.

**Comment by Theideabased** [comment]
i have corrected the  user class recheck again

File: models/user.py [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+     empty string for email
+     empty string for password
+     empty string for first_name
+     empty string for last_name
+     empty string for cohort """
+    email = ""
+    password = ""
+    first_name = ""
+    last_name = ""
+    cohort = int
```
Comment (KayFrimp) [inline models/user.py:17]: What is this man?

**Review by KayFrimp — CHANGES_REQUESTED** [review]

**Comment by Theideabased** [comment]
I have corrected the none user class value to the right syntax

**Pull request merged**